public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -282,6 +407,20 @@
         return ResponseEntity.ok(resp);
     }
 
+    /**
+     * This method gets concepts with maps to relationship in concept relationship table
+     *
+     * @param conceptId
+     * @return
+     */
+    @Override
+    public ResponseEntity<ConceptListResponse> getChildConceptsWithSourceCountGTZero(Long conceptId) {
```

Comment:
Nit: See my comment on the dao function this uses -- Can we have one function that takes an optional minSourceCount parameter that defaults to 0 ? 

Also, getSourceConcepts() is a better name. Child is not accurate actually. Children are something different I now know .  I just named the original wrong initially in my ignorance.